### PR TITLE
AADe, when I change a file in server/* folder the server is restarted

### PR DESCRIPTION
**Pull request by Miniplop**
Add  the hot reload for the SM Api

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Comment by Miniplop** [comment]
image

File: packages/slice-machine/package.json [hunk 1]
```diff
@@ -15,8 +15,7 @@
     "postinstall": "node postinstall",
     "prepublishOnly": "yarn bundle",
     "serve": "node -r esm ./build/server/src/index.js",
-    "watch-server": "tsc -w --project server/tsconfig.json",
-    "dev-server": "cross-env PORT=9999 CWD=./tests/project node build/server/src/index.js",
+    "dev-server": "cross-env PORT=9999 CWD=./tests/project ./node_modules/nodemon/bin/nodemon.js -e ts,js --watch server --ignore '*.test.ts' --exec ts-node server/src/index.js",
```
Comment (hypervillain) [inline packages/slice-machine/package.json]: Do you have to provide `./node_modules/nodemon/bin/nodemon.js` instead of `nodemon`?

Comment (Miniplop) [reply packages/slice-machine/package.json]: Good question, I think we can use nodemon directly

Comment (Miniplop) [reply packages/slice-machine/package.json]: Done

Comment (hypervillain) [reply packages/slice-machine/package.json]: thank you!

**Review by hypervillain — APPROVED** [review]

**Pull request merged**